airbyte-integrations/connectors/source-mysql-v2/src/test/kotlin/io/airbyte/integrations/source/mysql/MysqlCdcIntegrationTest.kt

```diff
@@ -4,253 +4,113 @@
-    }
-  },
-  "streamStates": [
-    {
-      "streamDescriptor": {
-        "name": "newcdk_table",
-        "namespace": "newcdk"
-      },
-      "streamState": {
-        "stream_name": "newcdk_table",
-        "cursor_field": [
-          "_ab_cdc_cursor"
-        ],
-        "stream_namespace": "newcdk"
-      }
+        @JvmStatic
+        @BeforeAll
+        @Timeout(value = 300)
+        fun startAndProvisionTestContainer() {
```

Comment:
should this be shared in test folder so other tests if we may have could use it? Or do you prefer to have all integration test added in this file?

Reply:
I'd prefer the latter, but we'll have to see if it's actually possible for all of them to share the same database instance or not. If it's not possible, then yes, this should be shared.